javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,45 @@
+const width = 12;
+const height = 14;
+const area = width * height;
+
+console.log('Value of the area:', area);
+console.log('Type of the area:', typeof area);
+
+const bill = 199;
+const payment = 50;
+const change = bill - payment;
```

Comment:
Your `payment` should be larger than the `bill`, and then your operands on line 10 need to be swapped

Reply:
👍🏽

Reply:
<img width="1502" alt="Screenshot 2023-10-23 at 3 14 09 PM" src="https://github.com/tiffanydbrown/c1023-code-solutions/assets/139157669/0c735197-2731-462f-8f95-02834944ab42">